src/call/github-workflow-runs-monitor.ts

```diff
@@ -70,4 +81,35 @@ export default async function githubWorkflowRunsMonitor(app: Probot, context: an
   } catch (error) {
     app.log.error(`Error indexing log data: ${error}`);
   }
+
+  let count: any;
```

Comment:
```suggestion
  let count: number;
```

Reply:
For cloudwatch I see `Value: number | undefined`, if i change to number I get the error `Variable count is used before being assigned.`

Reply:
Then initialize the value to be 0 or 1 to start with.

Reply:
Let me check this.

Reply:
I have started with 0 and add if for `failure` or `startup_failure`, please check.